src/pages/linux/index.md

```diff
@@ -5,6 +5,6 @@ title: Linux
 
 Linux is the most popular operating system in the world. It powers 90% of the world's servers, and is the basis of Google's popular Android operating system.
 
-There are also versions of Linux designed to be run on personal computers, such as Ubuntu.
+There are also many versions of Linux designed to be run on personal computers.
 
-For more facts about Ubuntu, read Quincy Larson's <a href='https://medium.freecodecamp.org/linux-is-25-yay-lets-celebrate-with-25-rad-facts-about-linux-c8d8ac30076d' target='_blank' rel='nofollow'>"Linux is 25. Yay! Let’s celebrate with 25 stunning facts about Linux"</a>
\ No newline at end of file
+For more facts about Linux, read Quincy Larson's <a href='https://medium.freecodecamp.org/linux-is-25-yay-lets-celebrate-with-25-rad-facts-about-linux-c8d8ac30076d' target='_blank' rel='nofollow'>"Linux is 25. Yay! Let’s celebrate with 25 stunning facts about Linux"</a>
```

Comment:
I think this is a good change, this post isn't about Ubuntu specifically, so it shouldn't say so. 👍